zerver/views/users.py

```diff
@@ -115,8 +115,9 @@ def deactivate_user_backend(
     user_profile: UserProfile,
     *,
     user_id: PathOnly[int],
-    deactivation_notification_comment: Annotated[str, StringConstraints(max_length=2000)]
-    | None = None,
+    deactivation_notification_comment: (
```

Comment:
I think you can move all the changes in this file to the first commit. The changes in the first commit would group together nicely up to this point.

If you need to add more changes here that are unrelated to the first commit, you can add a `# TODO` comment in that commit describing what you plan to do next. Then, in the next commit, you can replace that comment with the actual changes.